modules/installation-vsphere-config-yaml.adoc

```diff
@@ -103,48 +103,24 @@ imageContentSources: <18>
+<3> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
 +
 [IMPORTANT]
 ====
-If you disable simultaneous multithreading, ensure that your capacity planning
-accounts for the dramatically decreased machine performance.
-Your machines must use at least 8 CPUs and 32 GB of RAM if you disable
-simultaneous multithreading.
+If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Your machines must use at least 8 CPUs and 32 GB of RAM if you disable simultaneous multithreading.
 ====
-<4> You must set the value of the `replicas` parameter to `0`. This parameter
-controls the number of workers that the cluster creates and manages for you,
-which are functions that the cluster does not perform when you
-use user-provisioned infrastructure. You must manually deploy worker
-machines for the cluster to use before you finish installing {product-title}.
-<5> The number of control plane machines that you add to the cluster. Because
-the cluster uses this values as the number of etcd endpoints in the cluster, the
-value must match the number of control plane machines that you deploy.
+<4> You must set the value of the `replicas` parameter to `0`. This parameter controls the number of workers that the cluster creates and manages for you, which are functions that the cluster does not perform when you use user-provisioned infrastructure. You must manually deploy worker machines for the cluster to use before you finish installing {product-title}.
+<5> The number of control plane machines that you add to the cluster. Because the cluster uses this values as the number of etcd endpoints in the cluster, the value must match the number of control plane machines that you deploy.
 <6> The cluster name that you specified in your DNS records.
 <7> The fully-qualified hostname or IP address of the vCenter server.
-<8> The name of the user for accessing the server. This user must have at least
-the roles and privileges that are required for
-link:https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/master/documentation/vcp-roles.md[static or dynamic persistent volume provisioning]
-in vSphere.
+<8> The name of the user for accessing the server.
```

Comment:
is there a reason why the additional information with the link was removed here?